cucumber.js

```diff
@@ -1,12 +1,24 @@
+const path = require('path')
+const { config } = require('./tests/e2e/config')
+const fs = require('fs')
+
+if (!fs.existsSync(config.reportDir)) {
+  fs.mkdirSync(path.join(config.reportDir, 'cucumber'), { recursive: true })
+}
+
 module.exports = {
-  smoke: `
-  --require ./tests/smoke/**/*.ts
+  e2e: `
+  --require ./tests/e2e/**/*.ts
+  --retry 1
```

Comment:
+1 :)

Reply:
i want to collect metrics over time to see which parts are flaky, recording videos, traces... etc. is a heavy task which slows down things. we now automatically torn this on in case of a failure: https://github.com/owncloud/web/blob/01b477166571fb633ff543f8618ab77caaa44882/tests/e2e/cucumber/environment/index.ts#L96 which then needs at least one retry